starboard/shared/win32/video_decoder.cc

```diff
@@ -191,6 +193,15 @@ VideoDecoder::VideoDecoder(
       is_hdr_supported_(is_hdr_supported) {
   SB_DCHECK(output_mode == kSbPlayerOutputModeDecodeToTexture);
 
+  const int kMaxNameSize = 256;
+  char value[kMaxNameSize];
+  bool result = SbSystemGetProperty(kSbSystemPropertyModelName, value, kMaxNameSize);
```

Comment:
Can we reuse starboard::shared::uwp::GetXboxType()?

Reply:
Done

Reply:
Using ::starboard::shared::uwp::GetXboxType() causes build issue for win32 project. Should I add refences to uwp name space  in the win32 project or it is bate to move ::starboard::shared::uwp::GetXboxType() to some common namespace?

Reply:
We've verified that proposed reduction of the queue length doesn't affect Cobalt performance in both SDR and HDR modes. The number of frame drops is not changed for 30 and 60 fps videos.